cobra/mit/access.py

```diff
@@ -125,12 +126,15 @@ def lookupByClass(self, classNames, parentDn=None, propFilter=None):
             dnQuery = DnQuery(parentDn)
             dnQuery.classFilter = classNames
             dnQuery.queryTarget = 'subtree'
-            if propFilter:
-                dnQuery.propFilter = propFilter
+            self.__setQueryParams(dnQuery, kwargs)
             mos = self.query(dnQuery)
         else:
             classQuery = ClassQuery(classNames)
-            if propFilter:
-                classQuery.propFilter = propFilter
+            self.__setQueryParams(classQuery, kwargs)
             mos = self.query(classQuery)
         return mos
+
+        def __setQueryParams(self, query, queryParams):
```

Comment:
landscape.io is going to throw a warning on this method because of a lack of documentation.  Previously we were not very strict on this point for internal methods.  After thinking about it some more, it makes sense to document them if anything for our own sanity.  So please add a doc string for this.  Even something as simple as:

    """Set the query parameters for the query.
    
    An internal method to set query parameters on a query.
    
    Args:
      query (cobra.mit.request.AbstractQuery): The query object.
      queryParams (dict): The query parameters to set
     """